modules/installation-launching-installer.adoc

```diff
@@ -336,7 +336,7 @@ https://ovirtlab.example.com/ovirt-engine/api
 ----
 endif::openshift-origin[]
 +
-.. The installer automatically generates a CA certificate. For `Would you like to use the above certificate to connect to the Engine?`, answer `y` or `N`. If you answer `N`, you must install {product-title} in insecure mode.
+.. The installer automatically generates a CA certificate. For `Would you like to use the above certificate to connect to the {rh-virtualization-engine-name}?`, answer `y` or `N`. If you answer `N`, you must install {product-title} in insecure mode.
```

Comment:
Is this supposed to appear in _[Installing a cluster on RHV with customizations](https://deploy-preview-35508--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_rhv/installing-rhv-customizations.html#installation-launching-installer_installing-rhv-customizations)_? I'm not seeing it in the preview or [live docs](https://docs.openshift.com/container-platform/4.8/installing/installing_rhv/installing-rhv-customizations.html#installation-launching-installer_installing-rhv-customizations). I see these steps in the default RHV install.

Reply:
Does the code switch between `Engine` and `Manager`?

Reply:
@mgold1234 Can you help here?